### PR TITLE
Workaround sscg DH parameter generation bug in 8.7. (closes #171)

### LINKED ISSUE #171: Latest httpd-container (ubi8/httpd-24:1-230) stopped working on OCP by default
Just noticed this today on my OCP clusters (both 4.10.x and 4.11.x) when running a container with the image: registry.access.redhat.com/ubi8/httpd-24

Error:
```
[ec2-user@bastion multicloud-gitops]$ oc logs -n config-demo config-demo-c4995556-7nbzf
=> sourcing 10-set-mpm.sh ...
=> sourcing 20-copy-config.sh ...
=> sourcing 40-ssl-certs.sh ...
---> Generating SSL key pair for httpd...
Could not write to /opt/app-root/src/dhparams.pem. Check directory permissions.
```

The previous version (1-226 works fine, I just tested it). I assume https://github.com/sclorg/httpd-container/commit/5be03efe3bfc6bae74d88830f7cd09417a93e172 is somewhat related. No big deal I guess, but I thought I'd at least mention it as it might affect others.  The full deployment yaml we use is here https://github.com/hybrid-cloud-patterns/multicloud-gitops/blob/main/charts/all/config-demo/templates/config-demo-deployment.yaml

### PR DESCRIPTION
**Pull request by notroj**

### CODE REVIEW

**Review by uhliarik — COMMENTED** [review]
Change LGTM. dhparams file is created and stored in the directory with other TLS files if sscg contains this option.

File: 2.4-micro/root/usr/share/container-scripts/httpd/common.sh [hunk 1]
```diff
@@ -24,13 +24,19 @@ gen_ssl_certs() {
 
   echo "---> Generating SSL key pair for httpd..."
   if [ -x "/usr/bin/sscg" ]; then
+      if sscg --help | grep -q dhparams-file; then
```
Comment (uhliarik) [inline 2.4-micro/root/usr/share/container-scripts/httpd/common.sh:27]: Don't see any problem. Maybe only this indentation :).

Comment (notroj) [reply 2.4-micro/root/usr/share/container-scripts/httpd/common.sh]: Thanks @uhliarik - pushed a fix.

**Comment by phracek** [comment]
[test-all]

**Comment by notroj** [comment]
@phracek can I ignore the failures there?

**Pull request merged**